File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/delivery/model/dto/ShippingScheduleDto.java
package com.ccadmin.app.delivery.model.dto;

import java.util.ArrayList;
import java.util.List;

public class ShippingScheduleDto {

    public String ScheduleType;
    public String ScheduleName;
    public String UseTimeSlot;
    public List<ShippingScheduleDateDto> DateList = new ArrayList<>();
}

File: frontend/web_ccadmin_ecommerce/src/app/enterprise/client/model/dto/ShippingScheduleDto.ts
export class ShippingScheduleTimeSlotDto {
  public Label: string = '';
  public StartTime: string = '';
  public EndTime: string = '';
  public ScheduledFrom: string = '';
  public ScheduledTo: string = '';
}

export class ShippingScheduleDateDto {
  public Date: string = '';
  public Label: string = '';
  public ScheduledFrom: string = '';
  public ScheduledTo: string = '';
  public TimeSlotList: ShippingScheduleTimeSlotDto[] = [];
}

export class ShippingScheduleDto {
  public ScheduleType: string = '';
  public ScheduleName: string = '';
  public UseTimeSlot: string = 'N';
  public DateList: ShippingScheduleDateDto[] = [];
}
